### ISSUE TITLE
Errore Install

### ISSUE THREAD

**Issue by Fafofu** [ACCORD5/TrellisDesk#5]
TD Fatal Error: Cache - Cache directory not found: C:\inetpub\helpdesk\install\/data/cache/trellis/ in C:\inetpub\helpdesk\includes\classes\class_cache.php on line 26

IIS 7

**Comment by djtarazona** [comment]
How did you download your copy of Trellis Desk?  Via Github?

**Comment by Fafofu** [comment]
https://github.com/ACCORD5/TrellisDesk/

**Comment by djtarazona** [comment]
Directory should definitely be there.  Will look into it!

**Comment by Fafofu** [comment]
C:\inetpub\helpdesk\install\/data/cache/trellis/
C:\inetpub\helpdesk\install -> "\ /" <- data/cache/trellis/
LOOK!

**Comment by Fafofu** [comment]
I'm sorry, I'm from Russia and I use a translator =)

**Comment by djtarazona** [comment]
Yes, however PHP should be smart enough to handle that directory separator difference (I believe).  Which is why I still will be looking into the issue. :P

**Comment by Fafofu** [comment]
PHP smart IIS no :D

**Comment by DanielRuf** [comment]
you should set in the `config.php` file manually the `data_path` and `cache_path`

**Comment by Fafofu** [comment]
Well, I'll try.

**Comment by DanielRuf** [comment]
so it should be there something like `data\cache\trellis\` instead of `/data/cache/trellis/`  without the beginning forward slash and all forward slashes to backward slashes

**Comment by Fafofu** [comment]
And where is config.php?

In root only:
<?php

$config['pass_key'] = 'your unique password key phrase goes here';
$config['cookie_key'] = 'your unique cookie key phrase goes here';
$config['session_key'] = 'your unique session key phrase goes here';
$config['rss_key'] = 'your unique rss key phrase goes here';
$config['flatfile_key'] = '';

$config['debug_level'] = 0;

?>

**Comment by Fafofu** [comment]
If I change in config.php after install:

$config['hd_url'] = 'http://help.shvidler.local';
$config['data_path'] = 'C:\inetpub\helpdesk\install\data\';
$config['cache_path'] = $config['data_path'] .'cache\';
$config['logs_path'] = $config['data_path'] .'logs\';
$config['skin_compile_path'] = $config['data_path'] .'skin_compile\';
$config['temp_path'] = $config['data_path'] .'temp\';

"500 - Internal server error.

The problem with the requested resource, the resource can not be displayed."

**Comment by DanielRuf** [comment]
sorry, my fault. thought you use the latest version from me. this is the very last version

https://github.com/KAOSFORGE/TrellisDesk/blob/master/config.php.dist

you can add also `$config['data_path'] = '...'` there with the right value

TD first looks in the config.php for the correspinding variable and then if it is generated somewhere else

**Comment by DanielRuf** [comment]
print out the `$config['cache_path']`with `echo $config['cache_path']`

**Comment by Fafofu** [comment]
I do not understand: (

**Comment by Fafofu** [comment]
File & Folder Permissions
Configuration File Passed 
Cache Folders Failed 
Trellis Desk does not have permission to write to C:\inetpub\helpdesk\install/data/cache. Please CHMOD this folder to 0777. 
Data Folder Failed 
Trellis Desk does not have permission to write to C:\inetpub\helpdesk\install/data. Please CHMOD this folder to 0777. 
Languages Folder Passed 
Logs Folder Failed 
Trellis Desk does not have permission to write to C:\inetpub\helpdesk\install/data/logs. Please CHMOD this folder to 0777. 
Skins Folder Passed 
Skin Compile Folder Failed 
Trellis Desk does not have permission to write to C:\inetpub\helpdesk\install/data/skin_compile. Please CHMOD this folder to 0777. 
Temp Folder Failed 
Trellis Desk does not have permission to write to C:\inetpub\helpdesk\install/data/temp. Please CHMOD this folder to 0777. 
Uploads Folder Failed 
Trellis Desk does not have permission to write to C:\inetpub\helpdesk\install/data/uploads. Please CHMOD this folder to 0777.

**Comment by DanielRuf** [comment]
it is `C:\inetpub\helpdesk\data\` and not `C:\inetpub\helpdesk\install\data\`

**Comment by Fafofu** [comment]
Yes, I saw all the same error 500.

**Comment by Fafofu** [comment]
As soon as I though I was in one place "/" to "\" error 500.

If I use this config:

<?php

$config['pass_key'] = 'your unique password key phrase goes here';
$config['cookie_key'] = 'your unique cookie key phrase goes here';
$config['session_key'] = 'your unique session key phrase goes here';
$config['rss_key'] = 'your unique rss key phrase goes here';
$config['flatfile_key'] = '';
$data_path=getcwd();
$config['data_path'] = $data_path.'/data/';

$config['debug_level'] = 0;

?>

I do not get it to install.:

https://lh3.googleusercontent.com/-P3VarKAtqRs/Tp6gGLZ04QI/AAAAAAAABiA/vG8JWYoHhI4/s800/Trellis%252520Desk%252520Install%252520Center.htm_20111019140110.png

https://lh5.googleusercontent.com/-hDDEpBAIBe8/Tp6g233HATI/AAAAAAAABiM/vdNTzrptYeQ/s800/321.png

**Comment by DanielRuf** [comment]
Please just edit your comments. Please change the paths to the correct ones.
The `data` dir is not in the `install` dir, it is in the `helpdeskdir` on your machine?

**Comment by dhanasekar** [comment]
TD Fatal Error: Cache - Cache directory not found: E:\HostingSpaces\emsplus.in\emsplus.in\wwwroot\trell\helpdesk\install\/data/cache/trellis/ in

pl  reply how to solve this issue

**Comment by DanielRuf** [comment]
did you read all comments and please use also my last version as this here is outdated: https://github.com/KAOSFORGE/TrellisDesk

Seems you are on an IIS / windows machine?

**Comment by dhanasekar** [comment]
thanks   KAOSFORGE will install and say about the functioning..and i have one more doubt, that i tryed to install our trellis desk on my UK server... i got warning like  this

 Warning: date_default_timezone_get() [function.date-default-timezone-get]: It is not safe to rely on the system's timezone settings. You are _required_ to use the date.timezone setting or the date_default_timezone_set() function. In case you used any of those methods and you are still getting this warning, you most likely misspelled the timezone identifier. We selected 'Europe/London' for '0.0/no DST' instead in C:\inetpub\vhosts\dcsglobalinfo.com\httpdocs\Helpdesk\trell\index.php on line 26

> TD Fatal Error: Cache - Cache directory not found: trellis/ in C:\inetpub\vhosts\domain .com\httpdocs\Helpdesk\trell\includes\classes\class_cache.php on line 26

whether this new version will help to resolve that issue? we are using iis server

thanks in advance....

**Comment by DanielRuf** [comment]
Well i can check that if you use my latest version and not the one from accord5

Then i can see if these errors also occur.

Also please dont forget, there are maybe some issues with iis so currently trellis desk should run on a linux machine maybe.
